src/messages/responses/list-caches/list-caches.ts

```diff
@@ -0,0 +1,54 @@
+import {ListCachesResponse} from './list-caches-response';
+import {CacheInfo} from '../../cache-info';
+import {control} from '@gomomento/generated-types';
+import {MomentoErrorCode, SdkError} from '../../../errors/errors';
+
+export class Success extends ListCachesResponse {
+  private readonly nextToken: string | null;
+  private readonly caches: CacheInfo[];
+  constructor(result?: control.control_client._ListCachesResponse) {
+    super();
+    this.nextToken = result?.next_token || null;
+    this.caches = [];
+    result?.cache.forEach(cache =>
+      this.caches.push(new CacheInfo(cache.cache_name))
+    );
```

Comment:
Looks like a map?

Reply:
Yeah, it does. This was copied and pasted from the old response class. Kinda seems six one half dozen the other to me but if you have a compelling reason to change it I'll do it.

Reply:
Takes more time to figure out what's going on. And everywhere else uses map.

Reply:
ok, i changed it for you.

Reply:
+1 to map for this, thanks for changing it